HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -851,7 +851,7 @@ def from_oga(hpxml)
       @number_of_conditioned_floors_above_grade = to_integer_or_nil(XMLHelper.get_value(building_construction, 'NumberofConditionedFloorsAboveGrade'))
       @average_ceiling_height = to_float_or_nil(XMLHelper.get_value(building_construction, 'AverageCeilingHeight'))
       @number_of_bedrooms = to_integer_or_nil(XMLHelper.get_value(building_construction, 'NumberofBedrooms'))
-      @number_of_bathrooms = to_integer_or_nil(XMLHelper.get_value(building_construction, 'NumberofBathrooms'))
+      @number_of_bathrooms = to_float_or_nil(XMLHelper.get_value(building_construction, 'NumberofBathrooms'))
```

Comment:
This isn't correct. The [NumberofBathrooms element in HPXML is an integer](https://hpxml.nrel.gov/datadictionary/3.0.0/Building/BuildingDetails/BuildingSummary/BuildingConstruction/NumberofBathrooms). That's probably why we had some of the code structured the way it was.

Reply:
I realized the data type of NumberofBathrooms is defined as an integer in HPXML Toolbox 🤦‍♂️.  I reverted the datatype of NumberofBathrooms to an integer in the last push.